bodo/pandas/series.py

```diff
@@ -18,6 +21,37 @@ class BodoSeries(pd.Series, BodoLazyWrapper):
     _head_s: pd.Series | None = None
     _name: Hashable = None
 
+    def __init__(
+        self,
+        data=None,
+        index=None,
+        dtype: Dtype | None = None,
+        name=None,
+        copy: bool | None = None,
+        fastpath: bool | lib.NoDefault = lib.no_default,
+        plan: plan_operators.PlanOperator | None = None,
+    ) -> None:
+        self.__dict__["plan"] = plan
+        super().__init__(data, index, dtype, name, copy, fastpath)
+
+    def _cmp_method(self, other, op):
+        if hasattr(self, "plan") and self.plan != None:
+            zero_size_self = self.iloc[:0]
+            if hasattr(other, "iloc"):
+                zero_size_other = other.iloc[:0]
+            else:
+                zero_size_other = other
+            if hasattr(other, "plan") and other.plan != None:
+                other = other.plan
+            new_metadata = zero_size_self._cmp_method(zero_size_other, op)
+            assert isinstance(new_metadata, pd.Series)
+            return BodoSeries(
+                new_metadata,
+                plan=plan_operators.BinaryOpPlanOperator(self.plan, other, op),
+            )
```

Comment:
This code is fine as a placeholder but may not be very robust. Let's add some comments for now at least about the cases and data types that are possible.